twoliter/src/cmd/publish.rs

```diff
@@ -0,0 +1,57 @@
+use crate::cargo_make::CargoMake;
+use crate::project;
+use crate::tools::{install_tools, tools_tempdir};
+use anyhow::Result;
+use clap::Parser;
+use std::path::PathBuf;
+
+#[derive(Debug, Parser)]
+pub(crate) enum PublishCommand {
+    Ami(PublishAmi),
+}
+
+impl PublishCommand {
+    pub(crate) async fn run(self) -> Result<()> {
+        match self {
+            PublishCommand::Ami(publish_ami) => publish_ami.run().await,
+        }
+    }
+}
+
+/// Publish a Bottlerocket variant image.
+#[derive(Debug, Parser)]
+pub(crate) struct PublishAmi {
```

Comment:
Do these args match the pubsys command that is being surfaced? The same concern applies in that I don't think we should reproduce the internal build system's environment variable interface.